### ISSUE TITLE
Menu with animation breaks position

### ISSUE THREAD

**Issue by rettimo** [szhsin/react-menu#1347]
React/React-dom version: 18.3.1
React-Menu version: 4.1.0

## Describe the bug

I used the [tailwind example](https://codesandbox.io/p/sandbox/react-menu-tailwindcss-0r1rvf?) as a basis for my menu and styling. It was later noticed that due to the scale animation, I was getting the wrong menu position after fully opening.

### Example 1:
 ```javascript
keyframes: {
    fadeIn: { from: { opacity: 0, transform: "scale(.95)" } },
    fadeOut: { to: { opacity: 0, transform: "scale(.95)" } },
},
```
<img width="606" alt="image" src="https://github.com/user-attachments/assets/e45d9dd1-1fdc-4ae8-a91b-c0f02e999275">

### Example 2: 
 ```javascript
keyframes: {
    fadeIn: { from: { opacity: 0, transform: "scale(.5)" } },
    fadeOut: { to: { opacity: 0, transform: "scale(.5)" } },
},
```
<img width="293" alt="image" src="https://github.com/user-attachments/assets/b0355c8d-9896-4e3f-a899-4a878cac66fa">

## To Reproduce

1. Use [tailwind example](https://codesandbox.io/p/sandbox/react-menu-tailwindcss-0r1rvf?) as a basis.
2. Set `align="center"` to the `Menu` component.
3. Change the scale to any value, or leave the original.
4. Watch the menu shuffle when using this animation.

## Expected behavior

The use of animations should not affect the menu position

**Comment by szhsin** [comment]
Hi @rettimo , thanks for raising the issue. I will be looking into a possible fix of it.

**Comment by rettimo** [comment]
Any updates about this?

**Comment by szhsin** [comment]
Issue fixed by [v4.2.0](https://github.com/szhsin/react-menu/releases/tag/v4.2.0). 

This release also added a zooming animation css file which is used by default on the docs website: https://szhsin.github.io/react-menu/
